azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+# Pipeline for the TeamUp's webApp
+
+trigger:
```

Comment:
delete two previuous lines, let's make this the first line of the yml

Reply:
done